src/webui/www/private/download.html

```diff
@@ -191,6 +192,8 @@ <h2 class="vcenter">QBT_TR(Download Torrents from their URLs or Magnet links)QBT
             if (submitted)
                 window.parent.qBittorrent.Client.closeWindows();
         });
+
+        attachPathAutofill();
```

Comment:
I would prefer you don't use `class` and let this function accept a string/id parameter and call `addEventListener` on it.

Reply:
Could you tell me what is the reasoning behind avoiding use of classes for adding event listeners?      
Do you want to remove hidden logic from `attachPathAutofill()` linked with using class names defined inside of this function?

Reply:
>I would prefer you don't use class and let this function accept a string/id parameter and call addEventListener on it.

On a second thought, if there is going to be more than a few (>3) of components that will be using this autofill then it make sense to use `class`.